### ISSUE TITLE
[Enhancement] Add information about the last page refresh

### ISSUE THREAD

**Issue by Ibochkarev** [modxorg/DocsApp#9]
This will help you see how fresh the documentation is.
It would also be nice to add information - who updated the page.

**The example in the screenshot:**
![Starting a project on GitBook - GitBook 2019-03-26 18-45-32](https://user-images.githubusercontent.com/2138260/54998269-ec362a80-4ff7-11e9-92e8-b68610dc73ad.png)

**Comment by Mark-H** [comment]
Note to self, can be extracted with:

```
$ git --no-pager log --pretty=format:"%H | %aN | %aE | %ct | %s"  -- en/index.md 

1a3def3cdd673b20bf1933ad8844b90a0021b8ca | Ivan Bochkarev | ivanx86@gmail.com | 1556394446 | Update translation of the add-on homepage RU
0fe0f1aa2c686975c9574d78fc290b5e49f26d55 | Mark Hamstra | hello@markhamstra.com | 1555582373 | Add description for testing
9862d72612d2ee30e9643e4b2a96870c5ca809fd | Mark Hamstra | hello@markhamstra.com | 1555536757 | Test autodeployment
616f7c8867eaa1ad337656dbc04e832ec9e7d7c2 | Mark Hamstra | hello@markhamstra.com | 1554485609 | Update homepage to clarify the docs are about revo, with information on finding evo
d952b9b7cd2645455c276a9eb0aadcda24a281cf | Bochkarev Ivan | ivanx86@gmail.com | 1552688630 | Link fix after the reorganization
e242d0b0838bc831f75e3f35470b21062150e2bf | Bochkarev Ivan | ivanx86@gmail.com | 1552341739 | Finish fix markup
6b8333a216e7745766f5dbef19c47394ac6a76e2 | Bochkarev Ivan | ivanx86@gmail.com | 1552314387 | Add new translate
2e9287ef9e3eee316b47fb28ce30032b3c02bbf6 | Bochkarev Ivan | ivanx86@gmail.com | 1551813424 | Fix markup
350e524bfc369ba0dd27a1e3fe41172e501978aa | Mark Hamstra | hello@markhamstra.com | 1522342611 | Add a table converter to convert html tables into markdown tables(fixes #3)
06041eb13d8b591e49fc7b39022a3919651bb129 | Mark Hamstra | hello@markhamstra.com | 1522275683 | Add a tweak for demonstration purposes
98f6e5372efe5affe9679af250694034f04a71c1 | Mark Hamstra | hello@markhamstra.com | 1522274130 | Fix xpdo links
8a1324b91d267efe88071057c6930efdd2d05bf4 | Mark Hamstra | hello@markhamstra.com | 1518830672 | Slightly different markdown style preference and filtering to hopefully fix many formatting issues automatically
d1a0b55b80e4ea792d6712fc57a3612e2dde94d0 | Mark Hamstra | hello@markhamstra.com | 1518818576 | Ignore extras, evo, and xpdo 1 documentation. Simplify structure into single root project.%      
```

**Comment by Mark-H** [comment]
How's this?

![Schermafbeelding 2021-02-25 om 21 14 20](https://user-images.githubusercontent.com/312944/109211386-72dc7f00-77ae-11eb-9d5d-96f76ab60226.jpg)
